camille/inputs/bazefetcher.py

```diff
@@ -0,0 +1,41 @@
+from pandas import DataFrame, Timestamp, read_json
```

Comment:
I strongly prefer `import pandas as pd`, and then using e.g `pd.read_json`. The "as pd" is standard naming for pandas, and doing it like that makes it clear wherever a function from pd is used that it is indeed from pd. I have already been confused when reading this code where the read_json came from :-s